openapi/index.yaml

```diff
@@ -399,23 +524,74 @@ definitions:
   Timestamp:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v7.3.0/openapi/definitions.yaml#/Timestamp"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/Timestamp"
   TimeToLiveSeconds:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v7.3.0/openapi/definitions.yaml#/TimeToLiveSeconds"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/TimeToLiveSeconds"
   MessageContent:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v7.3.0/openapi/definitions.yaml#/MessageContent"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/MessageContent"
   NewMessage:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v7.3.0/openapi/definitions.yaml#/NewMessage"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/NewMessage"
+  ServicePayload:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/ServicePayload"
+  Service:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/Service"
+  ServiceMetadata:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/ServiceMetadata"
+  ServiceScope:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/ServiceScope"
+  ServiceId:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/ServiceId"
+  ServiceName:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/ServiceName"
+  OrganizationName:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/OrganizationName"
+  DepartmentName:
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/servicepayload-definition/openapi/definitions.yaml#/DepartmentName"
+  SubscriptionKeys:
+    type: object
+    properties:
+      primary_key:
+        type: string
+      secondary_key:
+        type: string
+    required:
+      - primary_key
+      - secondary_key
+  ServiceWithSubscriptionKeys:
+    allOf:
+      - $ref: "#/definitions/Service"
+      - $ref: "#/definitions/SubscriptionKeys"
+  ServiceIdCollection:
+    type: object
+    properties:
+      items:
+        type: array
+        items:
+          $ref: "#/definitions/ServiceId"
+      page_size:
+        type: number
+    required:
+      - items
+      - page_size
```

Comment:
@gunzip @francescopersico: for pagination purpose, we have to implement pagination with new  cosmos' sdk.